request-subgraph-storage/.resources/wget.txt

```diff
@@ -0,0 +1 @@
+https://nodejs.org/dist/latest-v16.x/node-v16.20.2-linux-x64.tar.gz
```

Comment:
Node 16 is end-of-life. Can we easily upgrade this to use Node 18?

Reply:
Or perhaps even Node 20?

Reply:
18 or 20 shouldn't be an issue